examples/nodejs-client/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "hawk-tracer-client-app",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "dependencies": {
+    "@hawktracer/client": "https://github.com/amzn/hawktracer.git"
```

Comment:
Until this PR is merged, this can be tested with "https://github.com/beila/hawktracer.git#node_js_support".